tests/sentry/api/endpoints/test_organization_member_invite_index.py

```diff
@@ -0,0 +1,470 @@
+        data = {
+            "email": "bill@localhost",
+            "orgRole": "billing",
+            "teamRoles": [],
+        }
+
+        serializer = OrganizationMemberInviteRequestSerializer(context=context, data=data)
+        assert not serializer.is_valid()
+
+
+class OrganizationMemberInviteListTest(APITestCase):
+    endpoint = "sentry-api-0-organization-member-invite-index"
+
+    def setUp(self):
+        self.approved_invite = self.create_member_invite(
+            organization=self.organization, email="user1@email.com"
+        )
+        self.requested_invite = self.create_member_invite(
+            organization=self.organization,
+            email="user2@email.com",
+            invite_status=InviteStatus.REQUESTED_TO_BE_INVITED.value,
+        )
+
+    def test_simple(self):
+        # if requestor doesn't have org admin permissions, only list approved invites
+        user = self.create_user("mifu@email.com", username="mifu")
+        self.create_member(organization=self.organization, user=user)
+        self.login_as(user)
+
+        response = self.get_success_response(self.organization.slug)
+
+        assert len(response.data) == 1
+        assert response.data[0]["email"] == self.approved_invite.email
+        # make sure we don't serialize token
+        assert not response.data[0].get("token")
+
+    def test_staff(self):
+        self.login_as(self.user)
+
+        response = self.get_success_response(self.organization.slug)
+
+        assert len(response.data) == 2
+        assert response.data[0]["email"] == self.approved_invite.email
+        assert response.data[0]["inviteStatus"] == "approved"
+        assert response.data[1]["email"] == self.requested_invite.email
+        assert response.data[1]["inviteStatus"] == "requested_to_be_invited"
+
+    def test_org_owner(self):
+        user = self.create_user("supreme-mifu@email.com", username="powerful mifu")
+        self.create_member(organization=self.organization, user=user, role="owner")
+        self.login_as(user)
+
+        response = self.get_success_response(self.organization.slug)
+
+        assert len(response.data) == 2
+        assert response.data[0]["email"] == self.approved_invite.email
+        assert response.data[0]["inviteStatus"] == "approved"
+        assert response.data[1]["email"] == self.requested_invite.email
+        assert response.data[1]["inviteStatus"] == "requested_to_be_invited"
+
+
+class OrganizationMemberInvitePermissionRoleTest(APITestCase):
+    endpoint = "sentry-api-0-organization-member-invite-index"
+    method = "post"
+
+    def setUp(self):
+        self.login_as(self.user)
+
+    def invite_all_helper(self, role):
+        invite_roles = ["owner", "manager", "member"]
+
+        user = self.create_user("user@localhost")
+        member = self.create_member(user=user, organization=self.organization, role=role)
+        self.login_as(user=user)
+
+        self.organization.flags.disable_member_invite = True
```

Comment:
it would be helpful to add a comment that when `organization.flags.disable_member_invite` is `True`, only roles with `member:admin` can invite members (aka not `admin` or lower) -- i had to search to remind myself why lol